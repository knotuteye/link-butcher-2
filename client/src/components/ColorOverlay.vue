<template>
  <div class="overlay" id="overlay">
    <slot></slot>
  </div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator'

@Component
export default class ColorOverlay extends Vue {}
</script>

<style scoped>
div.overlay {
  min-height: 100vh;
  width: 100vw;
  background: #0c04165b;
}
</style>
