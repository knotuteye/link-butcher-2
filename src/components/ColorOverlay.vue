<template>
  <div class="overlay">
    <slot></slot>
  </div>
</template>

<script lang="ts">
import { Component, Prop, Vue } from 'vue-property-decorator'

@Component
export default class ColorOverlay extends Vue {}
</script>

<style scoped>
div.overlay {
  min-height: 100vh;
  width: 100vw;
  background-color: #6f16ffaa;
}
</style>
