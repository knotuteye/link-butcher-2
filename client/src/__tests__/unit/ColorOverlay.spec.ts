import { shallowMount } from '@vue/test-utils'
import ColorOverlay from '../../components/ColorOverlay.vue'

describe('ColorOverlay.vue', () => {
  test('is rendered as designed', () => {
    const wrapper = shallowMount(ColorOverlay)
    expect(wrapper.element).toMatchSnapshot()
  })
})
