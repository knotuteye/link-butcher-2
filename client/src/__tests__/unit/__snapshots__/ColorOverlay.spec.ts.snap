// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorOverlay.vue is rendered as designed 1`] = `
<div
  class="overlay"
  id="overlay"
/>
`;
